site/_includes/themes/apache/_navigation.html

```diff
@@ -49,7 +49,7 @@
     <h1>{{ site.data.project.name }}</h1>
     <p>{{ site.data.project.description }}</p>
 
-    {% if site.data.project.latest_release %}
+    {% if site.data.project.download %}
```

Comment:
this test should be for `spark_latest_release` and `flink_latest_release` (barring "TBA", of course)